README.rst

```diff
@@ -94,7 +94,7 @@ Contributing to Strawberry Fields
 =================================
 
 We welcome contributions - simply fork the Strawberry Fields repository, and then make a
-`pull request <https://help.github.com/articles/about-pull-requests/>`_ containing your contribution.  All contributers to Strawberry Fields will be listed as authors on the releases.
+`pull request <https://help.github.com/articles/about-pull-requests/>`_ containing your contribution.  All contributors to Strawberry Fields will be listed as authors on the releases.
```

Comment:
```suggestion
`pull request <https://help.github.com/articles/about-pull-requests/>`_ containing your
contribution.  All contributors to Strawberry Fields will be listed as authors on the releases.
```
Just to conform to 100 char line-width, but it looks like this doc doesn't enforce that so I don't mind.